assets/src/components/SelectCard.js

```diff
@@ -8,9 +8,7 @@ import Typography from '@material-ui/core/Typography'
 
 const styles = theme => ({
   card: {
-    width: 325,
-    display: 'flex',
-    margin: theme.spacing(1)
+    margin: '20px'
```

Comment:
Can you explain why we're moving away from `theme.spacing(1)` here?

Reply:
Good point, I changed it now to `theme.spacing(3)`

Reply:
Thoughts on bumping it down to `2`?

Reply:
I'm open to it, happy with either. Should I change it to 2?

Reply:
I like `2` better, but I could probably be convinced.